examples/demo_all_cisco.pp

```diff
@@ -33,9 +33,12 @@
   include ciscopuppet::cisco::demo_command_config
   include ciscopuppet::cisco::demo_dhcp_relay_global
   include ciscopuppet::cisco::demo_evpn
+  include ciscopuppet::cisco::demo_evpn_multisite
```

Comment:
This should only be including the single demo manifest now right?

Reply:
Yes. removed reference to the older manifests.